docs/Deploying-Pantheon/High-Availability.md

```diff
@@ -0,0 +1,102 @@
+description: Pantheon high availability 
+<!--- END of page meta data -->
+
+# High Availability 
+
+To enable high availability to the [RPC Pub/Sub API over WebSockets](../Pantheon-API/RPC-PubSub.md) 
+or the [JSON-RPC API](../Pantheon-API/Using-JSON-RPC-API.md) run and synchronize multiple Pantheon 
+nodes to the network. Use a load balancer to distribute requests across nodes in the cluster that 
+are ready to receive requests. 
+
+## Determining When a Node is Ready 
+
+To determine when a node is ready use: 
+
+* [`net_peerCount`](../Reference/Pantheon-API-Methods.md#net_peercount) to confirm the minimum number of peers has been reached
+* [`eth_syncing`](../Reference/Pantheon-API-Methods.md#eth_syncing) to confirm the node is within the required number of blocks 
+from the best known block for that node 
+
+!!! note
+    The minimum number of peers and number of blocks from the best known block for a node to be considered ready
+    is deployment specific. 
```

Comment:
With the release of 1.1.3, we can now recommend people use the /readiness endpoints for this instead of having to manually piece together these two JSON-RPC methods.